.mk/docker.mk

```diff
@@ -20,13 +20,19 @@ RUN_CONTAINERS=$(BUILD_CONTAINERS)
 KILL_CONTAINERS=$(BUILD_CONTAINERS)
 LOG_CONTAINERS=$(KILL_CONTAINERS)
 ORG=networkservicemesh
+ifeq ($(shell uname -m), x86_64)
+  VPP_AGENT=ligato/vpp-agent:v1.8.1
+endif
+ifeq ($(shell uname -m), aarch64)
```

Comment:
Can we define a global so I can override it if I decide to:
```Makefile
ARCH ?= $(shell uname -m)
```

Reply:
I modified according your suggestion